Apps/WebAPI/sdLitica.WebAPI/Models/Management/CreateUserModel.cs

```diff
@@ -0,0 +1,34 @@
+
+namespace sdLitica.WebAPI.Models.Management
+{
+    /// <summary>
+    /// This class represents unique identity and info of the user
+    /// </summary>
+    public class CreateUserModel
+    {
+        /// <summary>
+        /// User first name
+        /// </summary>
+        public string FirstName { get; set; }
+        /// <summary>
+        /// User last name
+        /// </summary>
+        public string LastName { get; set; }
+        /// <summary>
+        /// User email
+        /// </summary>
+        public string Email { get; set; }
```

Comment:
Maybe Email can has a "Required" attribute, since we used them to query db.

Reply:
Fixed and also added this mark to Password field